crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -152,6 +153,32 @@ impl<'a> Commands<'a> {
         });
     }
 
+    /// Inserts a [`crate::system::System`] into a [`crate::schedule::Stage`]
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// use bevy_ecs::prelude::*;
+    ///
+    /// fn another_system() {
+    ///    // A normal system like any other
+    /// }
+    ///
+    /// fn example_system(mut commands: Commands) {
+    ///     commands.insert_system(another_system, "some stage");
+    /// }
+    /// ```
```

Comment:
Most doctests we have also verify something about the example, like trying to turn it into a system, or even running it - often "offscreen", in a comment.